core/src/main/java/com/kin/ecosystem/core/data/blockchain/BlockchainSource.java

```diff
@@ -117,7 +119,9 @@ void sendTransaction(@NonNull String publicAddress, @NonNull BigDecimal amount,
 
 		void setActiveUserWallet(String kinUserId, String publicAddress);
 
-		void deleteAccountIndexKey();
+		void removeAccountIndexKey();
+
+		void clearCachedBalance();
```

Comment:
maybe call it log out if in the future there will be an additional properties that need to be cleared

Reply:
it the future we will split if needed it doesn't matter now.